hello-world-master/source/components/scene/getvideodata.brs

```diff
@@ -32,11 +32,16 @@ end function
 
 function populateItemContent(videoData as Object) as Object
     itemContent = CreateObject("roSGNode","ContentNode")
+    drmParams = {
+        keySystem: "Widevine"
+        licenseServerURL: "https://cwip-shaka-proxy.appspot.com/no_auth"
+    }
     itemContent.id = videoData.id
-    itemContent.title = videoData.title
+    itemContent.title = "DRM Video" 'itemContent.title = videoData.title
     itemContent.FHDPosterUrl = videoData.poster
-    itemContent.url = videoData.stream.url
-    itemContent.streamformat = videoData.stream.format
+    itemContent.url = "https://cdn.bitmovin.com/content/assets/art-of-motion_drm/mpds/11331.mpd" 'itemContent.url = videoData.stream.url
+    itemContent.streamformat = "dash" 'itemContent.streamformat = videoData.stream.format
+    itemContent.drmParams = drmParams
```

Comment:
I would sugest to separate the drm attribution and the previous data attribution
You can heep all the previous attributions commented under the current code not one at the end of another